plugins/inputs/snmp_trap/snmp_trap_test.go

```diff
@@ -0,0 +1,149 @@
+package snmp_trap
+
+//todo: tests that look up oids will pass only if snmptranslate (part
+//of net-snmp) is installed and working.  We need to mock name lookup
+//or add a way to disable it so tests will pass when snmptranslate
+//isn't available.
+
+import (
+	//"log"
+	//"os"
+	"net"
+	"testing"
+	"time"
+
+	"github.com/soniah/gosnmp"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/testutil"
+
+	"github.com/influxdata/telegraf/plugins/inputs/snmp"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTranslate(t *testing.T) {
+	mibName, oidNum, oidText, conversion, err := snmp.SnmpTranslate(".1.3.6.1.6.3.1.1.5.1")
```

Comment:
Is there a reason this test isn't in `inputs/snmp/snmp_test.go`, as it only tests a function from that package?

Reply:
I put it here so these tests fail quickly if somebody changes snmp.SnmpTranslate() without knowing that it's used by snmp_trap.